Sources/Plasma/Apps/plClient/Mac-Cocoa/main.mm

```diff
@@ -108,7 +108,7 @@ @interface AppDelegate : NSWindowController <NSApplicationDelegate,
    @public
     plClientLoader gClient;
     dispatch_source_t _displaySource;
-    plMacDisplayHelper _displayHelper;
+    plMacDisplayHelper *_displayHelper;
```

Comment:
```suggestion
    plMacDisplayHelper* _displayHelper;
```

What is happening?

Reply:
Clang-format didn't get this one either (possibly because I gave it multiple commits to work through.)

I had it do a full pass on this file - but it's giving me a lot of changes. I'll trim that change set down.